webapp/src/components/conference/conference.tsx

```diff
@@ -99,6 +99,10 @@ export default class Conference extends React.PureComponent<Props, State> {
             jwt: this.props.jwt,
             noSSL,
             parentNode: document.querySelector('#jitsiMeet'),
+            configOverwrite: {
+                startWithVideoMuted: true,
+                startWithAudioMuted: true
+            },
```

Comment:
Are these necessary for fixing the issue or is the issue just fixable by `https` change?

Reply:
No these are not necessary for the fix.....I added this to keep the camera and microphone off by default.

Reply:
I don't think there's a requirement for this. You can remove this.